index.html

```diff
@@ -153,6 +153,35 @@ <h1 id="title">Decentralized Identifier Working Group Charter</h1>
 (including, but not limited to, switching between Verifiable Credential digital wallet providers).
           </li>
         </ul>
+      <p>
```

Comment:
The more text a W3C Charter has, the more likely one of the 430+ W3C Members are to object to one of the sentences. W3C Charters are meant to be extremely sparse (some would say that the DID WG Charter already has way too much in it). 

The text below is good, we should put it in the DID spec. 

Suggesting that "DIDs ... lead to some complexity" will provide something to pounce on for the organizations that don't want to see this work go anywhere.